### PR TITLE
Fix/ci

### PR DESCRIPTION
**Pull request by mandresm**
trying to fix the CI with a new ubuntu container

### CODE REVIEW

**Comment by mandresm** [comment]
This is finally ready to be reviewed and merged.

File: .github/containers/ubuntu-miniforge-esm_tests/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+docker run -it --rm ubuntu-miniforge-esm_tests
+```
+
+To tag it and push it to Docker Hub:
+
+```bash
+docker tag ubuntu-miniforge-esm_tests mandresm/ubuntu-miniforge-esm_tests:v1.0
+docker login
+docker push mandresm/ubuntu-miniforge-esm_tests:v1.0
+```
```
Comment (pgierz) [inline .github/containers/ubuntu-miniforge-esm_tests/README.md:19]: Not urgent, but GitHub has it's own internal container registry: [ghcr.io](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry), we could host the test container there.

File: .github/workflows/esm_tests_computer_workflow.yml [hunk 2]
```diff
@@ -30,46 +30,32 @@ jobs:
       LC_ALL: en_US.UTF-8
       LANG: en_US.UTF-8
     container:
-      image: koldunovn/esm_test:latest
+      image: mandresm/ubuntu-miniforge-esm_tests:v1.0
```
Comment (pgierz) [inline .github/workflows/esm_tests_computer_workflow.yml:34]: Change to ghcr hosting at some point in the future.

**Review by pgierz — APPROVED** [review]
Just an optional idea, see below.

File: .github/containers/ubuntu-miniforge-esm_tests/README.md (hunk 1 above)
Comment (mandresm) [reply .github/containers/ubuntu-miniforge-esm_tests/README.md:19]: good to know

File: .github/containers/ubuntu-miniforge-esm_tests/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Use Ubuntu 25 as the base image
```
Comment (mandresm) [inline .github/containers/ubuntu-miniforge-esm_tests/Dockerfile:3]:
```suggestion
# Use Ubuntu 25 as the base image

# TODO (PG): Change to ghcr hosting at some point in the future.
```

File: .github/containers/ubuntu-miniforge-esm_tests/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+# TODO (PG): Change to ghcr hosting at some point in the future.
+FROM ubuntu:25.04
```
Comment (mandresm) [inline .github/containers/ubuntu-miniforge-esm_tests/Dockerfile]:
```suggestion
# TODO (PG): Change to ghcr hosting at some point in the future.

# Use Ubuntu 25 as the base image
FROM ubuntu:25.04
```

File: .github/workflows/esm_tests_computer_workflow.yml (hunk 2 above)
Comment (mandresm) [reply .github/workflows/esm_tests_computer_workflow.yml:34]:
```suggestion
      # TODO (PG): Change to ghcr hosting at some point in the future.
      image: mandresm/ubuntu-miniforge-esm_tests:v1.0
```

**Comment by mandresm** [comment]
#bump

I am bumping because I removed some conda channels from the `environment.yaml`

**Pull request merged**